apps/hash-external-services/docker-compose.yml

```diff
@@ -117,6 +117,28 @@ services:
     environment:
       COLLECTOR_OTLP_ENABLED: "true"
 
+  type-fetcher:
+    init: true
+    build:
+      dockerfile: docker/Dockerfile
+      context: ../../apps/hash-graph
+      args:
+        PROFILE: dev
+    environment:
+      HASH_GRAPH_LOG_FORMAT: "${HASH_GRAPH_LOG_FORMAT:-pretty}"
+      RUST_LOG: "${HASH_GRAPH_LOG_LEVEL:-info,hash_type_fetcher=trace,reqwest=debug}"
+      RUST_BACKTRACE: 1
+    entrypoint: "/hash-type-fetcher"
+    command: ["--type-fetcher-host", "0.0.0.0", "--type-fetcher-port", "4444"]
+    healthcheck:
+      test:
+        [
+          "CMD-SHELL",
+          "netstat -an | grep 4444 > /dev/null; if [ 0 != $? ]; then exit 1; fi",
+        ]
```

Comment:
I think we have at 3 types of TCP health checks in the codebase :D

I don't think it's worth it for us to try to simplify it, I just find it funny we have so many slightly different setups that require very different health checks

Reply:
I replaced the healthchecks in https://github.com/hashintel/hash/pull/2038